server/cmd/dcrdex/config.go

```diff
@@ -414,6 +417,23 @@ func loadConfig() (*dexConf, *procOpts, error) {
 		cfg.LogDir = filepath.Join(cfg.AppDataDir, cfg.LogDir)
 	}
 
+	if cfg.Logo == "" {
+		cfg.Logo = filepath.Join(cfg.AppDataDir, defaultLogoName)
+	} else if !filepath.IsAbs(cfg.Logo) {
+		cfg.Logo = filepath.Join(cfg.AppDataDir, cfg.Logo)
```

Comment:
Docs say 
```
logo=path/to/logo.png
```
 and don't mention that it will be interpreted as a subpath of `AppDataDir`.